elements/pfe-autocomplete/src/pfe-autocomplete.js

```diff
@@ -309,14 +309,23 @@ class PfeAutocomplete extends PFElement {
     this._closeDroplist();
   }
 
+  /**
+   * Returns the HTML of the active element
+   * @param {integer} activeIndex Index of an element in the droplist
```

Comment:
```suggestion
   * @param {number} activeIndex Index of an element in the droplist
```

Integer is not a valid JSDoc type so I would suggest using `number` here instead.

Reply:
@castastrophe thanks for catching that! I will get that updated.